packages/ckeditor5-ui/theme/globals/_poweredby.css

```diff
@@ -5,8 +5,8 @@
 
 :root {
 	--ck-powered-by-line-height: 10px;
-	--ck-powered-by-padding-top: 2px;
-	--ck-powered-by-padding-left: 4px;
+	--ck-powered-by-padding-vertical: 2px;
```

Comment:
What are these changes in the feature's CSS?

Reply:
@pszczesniak

Reply:
Yes, talked with @oleq that the names of those two CSS variables are not perfect, so i've change it here - to simplify it's all in this PR with "PoweredBy" guide. in short: `--ck-powered-by-padding-top` is used to set padding `top` and `bottom`, `--ck-powered-by-padding-left` is used to set padding `left` and `right`.